src/HTTP/Response.php

```diff
@@ -0,0 +1,197 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * SimplePie
```

Comment:
Since this is based on PSR code, we need to attribute it properly and mention the license:

https://github.com/php-fig/http-message/blob/efd67d1dc14a7ef4fc4e518e7dee91c271d524e4/LICENSE#L1

This was actually one of the reasons I suggested SPDX, then we could just plop at the top:

```php
// SPDX-FileCopyrightText: 2014 PHP Framework Interoperability Group
// SPDX-FileCopyrightText: 2023 Art4 <art4@wlabs.de>
// SPDX-License-Identifier: MIT
```

We would also need to include the MIT license file (perhaps copy it into the repo as `LICENSE.MIT.md`. We could remove it once we remove this compat code.

Reply:
I would keep this for know to #798 as long as no decision has been made on this.